lib/webpacker/configuration.rb

```diff
@@ -47,10 +47,10 @@ def fetch(key)
     end
 
     def data
-      if env.development?
-        refresh
-      else
+      if env.production?
```

Comment:
@dhh is right as I've seen many rails installs where there are various flavors of production using for staging and performance tests.